lab_1_classify_by_unigrams/main.py

```diff
@@ -15,11 +15,22 @@ def tokenize(text: str) -> list[str] | None:
     Args:
         text (str): A text
 
+
     Returns:
         list[str] | None: A list of lower-cased tokens without punctuation
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+
+    if isinstance(text, str):
```

Comment:
there is no need in this check. Why?

Reply:
i felt like it needed it, but i do understand now that it's useless
fixed